packages/docs/src/Toaster.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import {
+  Toaster,
+  ToasterContent,
+  ToasterTitle,
+  ToasterButton,
+  Text,
+  IconBox,
+  Flex,
+} from '@citric/core'
+import { InfoCircle, Times } from '@citric/icons'
+
+export default {
+  title: 'Components/Toaster',
+  subcomponents: { ToasterContent, ToasterTitle, ToasterButton },
+} as ComponentMeta<typeof Toaster>
+
+export const toaster: ComponentStory<typeof Toaster> = (props) => (
+  <Flex sx={{ m: 10 }}>
+    <Toaster {...props}>
```

Comment:
```suggestion
export const toaster = () => (
  <Flex sx={{ m: 10 }}>
    <Toaster>
```

Melhor remover essas definições, o código exibido no "show code" do Figma está incompreensível e ao testar a aplicação o Git mostra que existe um problema

https://github.com/stack-spot/citric/runs/5323285584?check_suite_focus=true